index.html

```diff
@@ -7,10 +7,68 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <script type="text/javascript">
+        function classToggle() {
+        const navs = document.querySelectorAll('.navbar_items')
```

Comment:
I would indent these two lines to the right to make it look like they are separate